irc-against-humanity.json

```diff
@@ -19,44 +19,45 @@
 	{"text": "You can never have too much %s."},
 	{"text": "I didn't know about %s until %s introduced me to it.", "pick": 2},
 	{"text": "%s and %s. I ship it.", "pick": 2},
-	{"text": "%s looked through my browser history, only to discover %s.", "pick": 2}
+	{"text": "%s looked through my browser history, only to discover %s.", "pick": 2},
+	{"text": "How to beat your children without leaving any sign of injury"}
 ],
 "answers": [
-	"Imaginary boyfriend",
+	"An imaginary boyfriend",
 	"Frolicking in a meadow",
 	"Your own mother",
```

Comment:
noita666 gave me it as "Your mom" but I thought "Your mother" fit, idk why it's NOW "Your own mother", unless I did that by mistake.